rules/windows/credential_access_wbadmin_ntds.toml

```diff
@@ -0,0 +1,83 @@
+[metadata]
+creation_date = "2024/06/05"
+integration = ["windows", "endpoint", "system"]
+maturity = "production"
+updated_date = "2024/06/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of wbadmin to access the NTDS.dit file in a domain controller. Attackers with privileges from
+groups like Backup Operators can abuse the utility to perform credential access and compromise the domain.
```

Comment:
Do these groups have common IDs that can be added to the query?

Reply:
Process events don't contain the groups that the user is a part of, but to be able to execute this action, the user needs to have the SeBackupPrivilege right assigned